cmd/axiom/ingest/ingest.go

```diff
@@ -97,11 +111,11 @@ func NewIngestCmd(f *cmdutil.Factory) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringSliceVarP(&opts.Filenames, "file", "f", nil, "File to ingest")
-	cmd.Flags().StringVar(&opts.TimestampField, "timestamp-field", "", "Field to take the ingestion time from")
-	cmd.Flags().StringVar(&opts.TimestampFormat, "timestamp-format", "", "Format the timestamp is formatted in")
+	cmd.Flags().StringSliceVarP(&opts.Filenames, "file", "f", nil, "File(s) to ingest (- to read from stdin). Compressed input not supported")
```

Comment:
`Compressed input not supported`

Sounds like we should get rid of the `--compression` option all the way?

Reply:
Yeah, I am not sure the is a sensible reason for ever switching off compression... But preferably remove the flag in another change set?

Reply:
created https://github.com/axiomhq/cli/issues/13